workspaces/announcements/plugins/announcements/src/index.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export { announcementsPlugin, AnnouncementsPage } from './plugin';
+
+import {
+  announcementsApiRef as announcementsApiRef_,
+  AnnouncementsApi as AnnouncementsApi_,
+} from '@procore-oss/backstage-plugin-announcements-react';
+
+/**
+ * @deprecated Use `AnnouncementsApi` from `@procore-oss/backstage-plugin-announcements-react` instead
```

Comment:
Migrating deprecation warnings 1:1. The plan is to remove all deprecations as part of a single changeset after migration is complete.

Reply:
OK, that works for me 👍